packages/devtools_app/lib/src/config_specific/drag_and_drop/drag_and_drop.dart

```diff
@@ -66,4 +174,64 @@ abstract class DragAndDropState extends State<DragAndDrop> {
   void drop() {
     _dragging.value = false;
   }
+
+  void activate(bool active) {
+    _isActive = active;
+  }
+}
+
+const _dragAndDropEventAbsorberId = 'DragAndDropEventAbsorber';
```

Comment:
Can't this just be the DragAndDrop widget with no handler on drag. Behavior could be that if you have no handler on drag that means nothing should be visually rendered on drag.

Reply:
done